test/razor/razorIntegrationTests/testAssets/RazorApp/Pages/Completion.razor

```diff
@@ -0,0 +1,10 @@
+@page "/completion"
+
+@* Insert text tag in below using completion *@
+@{
+    <te
```

Comment:
If the test is inserting `<te`, should this be here?

Reply:
nope. I don't know why my local change didn't stay... removing